src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          maxlength="25"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          How old are You?
+          <input
+          type="number"
+          name="How old are You?"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </div>
+
+        <div class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="Full date of birth"
+          >
+        </div>
+
+        <div class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="I accept the term of the agreement"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="E-mail"
+          placeholder="email@example.com"
+          >
+        </div>
+        <div class="form-field">
+          Password:
+          <input
+          type="password"
+          name="Password"
+          minlength="5"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="Yes"
```

Comment:
Radio buttons for the same question should have the same 'name' attribute. Change the 'name' attribute to 'loveCats' for both 'Yes' and 'No' options.